index.js

```diff
@@ -414,65 +414,94 @@ demo`;
+  login:    Logs in to your Crowdbotics account.
+  logout:   Logs out of your Crowdbotics account.
+  modules:  Manages modules for your organization.
+  parse:    Parses and validates module definitions.
+  publish:  Publishes your modules to your organization's private catalog. When run, this command prompts for more information.
+  remove:   Removes one or more modules from your Demo app.
+  upgrade:  Upgrades your existing app's scaffold to the latest version or a specific version.
+
+--------------------
+More details
+--------------------
+
+add: Installs one or more modules into your Demo app.
+    Arguments:
+        --source: (Optional) Specifies the source directory for modules (defaults to the Crowdbotics modules directory based on package location).  
+        --project: (Optional) Specifies the project directory to install modules into (defaults to the git root of the current working directory).
+    Parameters: 
+        <module1> <module2>: Names of the modules to install (required).
+    Example: cb add module1 module2
+
+commit: Updates the definition of an existing module.
+    Arguments:
+        --source: (Optional) Specifies the source directory for modules (defaults to the current working directory).
+    Parameters: 
+        <module1> <module2>: Names of the modules to update (required).
+    Example: cb commit module1
+
+create: Creates a new module of a specific type.
+    Arguments:
+        --target: (Optional) Specifies the target directory for creating the module (defaults to the modules directory within the git root).
+    Parameters:
+        --name <name>: Name for the new module (required).  
+        --type <type>: Type of module to create (e.g., react-native) (required).
+    Example: cb create --name myModule --type react-native
+
+init: Initializes a new blank modules repository.
+    Arguments:
+        --name <project-name>: Specifies the name of the new project directory to create.
+    Example: cb init --name my-modules
+
+login: Logs in to your Crowdbotics account.
+    Example: cb login
+
+logout: Logs out of your Crowdbotics account.
+    Example: cb logout
+
+modules: Manages modules for your organization.
+    Arguments:
+        --search: (Optional) Searches for modules by name or description.   
+        --visibility: (Optional) Filters modules by visibility (private or public). 
+        --status: (Optional) Filters modules by status (published or archived). 
+        --page: (Optional) Specifies the page number for paginated results. 
+        --unarchive: (Optional) Flag to unarchive an archived module.
+    Parameters:
+        list: Lists available modules (optional subcommand).
+        get <id>: Gets details for a specific module by its ID (required subcommand with parameter).
+        archive <id>: Archives a specific module by its ID (required subcommand with parameter).
+        help: Shows help for the modules command (optional subcommand).
+    Example: 
```

Comment:
we either need to remove this or generate some real examples here.  there are a lot of arguments and parameters here.